pre_commit_mirror_maker/all/.pre-commit-hooks.yaml

```diff
@@ -1,5 +1,6 @@
 -   id: {id}
     name: {name}
+    description: {description!r}
```

Comment:
I think this will break the website: https://github.com/pre-commit/pre-commit.com/blob/07565cb17214510aea9bb69fce5681ebe701372b/hooks.mako#L41-L42

Reply:
Ah. Would it be better to support an “if” here then? Could use a template engine - I guess you prefer mako?

Reply:
I kinda wanted to avoid a templating engine but it might be unavoidable -- I've gone back and forth between mako and https://github.com/Yelp/yelp_cheetah -- not really sure which I prefer at the moment

the other, probably simpler, alternative is to fix the website to ignore empty descriptions

Reply:
Okay if that's a legit approach I'll take a look.

Reply:
Website was fixed in https://github.com/pre-commit/pre-commit.com/pull/618